script/wrapdocker

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Adapted from https://github.com/jpetazzo/dind
```

Comment:
Why not just use the canonical https://github.com/docker/docker/blob/738bdec4908755761b27b7d15baeefb1a824f770/project/dind instead?